8/fpm-conf-templates/zz2-docker-custom.conf

```
@@ -5,3 +5,4 @@ pm.start_servers = ${PHP_FPM_START_SERVERS}
 pm.min_spare_servers = ${PHP_FPM_MIN_SPARE_SERVERS}
 pm.max_spare_servers = ${PHP_FPM_MAX_SPARE_SERVERS}
 pm.status_path = ${PHP_FPM_PM_STATUS_PATH}
+access.format = '%R - %u %t \"%m %r?%q\" %s'
```

```suggestion
access.format = '%R - %u %t \"%m %r%Q%q\" %s'
```

---

We have tested the following format:

```suggestion
access.format = '%t - %s "%m %r%Q%q" %{REQUEST_URI}e'
```

An example log row is:

```bash
27/Sep/2023:14:31:56 +0000 - 200 "GET /index.php?path=/my-awesome-path" /router/translate-path?path=/my-awesome-path
```